.github/workflows/docker.yml

```diff
@@ -12,6 +12,9 @@ on:
 env:
   IMAGE_NAME: ${{ github.repository }}/${{ github.event.repository.name }}
 
+permissions:
+  packages: write
+
```

Comment:
Can you please remove these three lines of permission, I have added them for the action "snok/container-retention-policy@v2" and seems to have forgotten to remove them back from this repo.

Reply:
Removed permission lines